### PR TITLE
feature/Add speech to text typo

### PR DESCRIPTION
**Pull request by Prikshit7766**
## Checklist:

- [ ] I've added Google style docstrings to my code.
- [ ] I've used `pydantic` for typing when/where necessary.
- [ ] I have linted my code
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by Prikshit7766** [comment]
time complexity issue
Now it is taking 
10- 12 minutes to generate the records for classification task  [Probelm]
1-2 minutes to generate the records for ner ,qa , summarization

**Comment by chakravarthik27** [comment]
Hey @Prikshit7766,
can you please resolve the conflicts by merging release/1.4.0

File: nlptest/utils/SoundsLikeFunctions.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        if text in self.cmu:
+            return self.cmu[text][0]
+        elif text in self.CMU_dict:
+            return self.CMU_dict[text]
+        else:
+
+            return prons
+g2p = G2p()
+        
+class Word_Functions():
```
Comment (chakravarthik27) [inline nlptest/utils/SoundsLikeFunctions.py:24]: remove brackets after the class name.

**Word_Functions()**: -> **Word_Functions**:

do the same thing for classes except those are inherited from other types.

**Review by chakravarthik27 — APPROVED** [review]

File: nlptest/transform/robustness.py [hunk 2]
```diff
@@ -889,3 +890,81 @@ def insert_abbreviation(text):
...
+                        converted_sentence.append(word)
+
+            perturbed_text = ''.join(converted_sentence)
+            sample.category = "robustness"
+            if sample.task in ("ner", "text-classification"):
+                sample.transformations = transformations
+            
+            return perturbed_text
+
+        for sample in sample_list:
```
Comment (chakravarthik27) [inline nlptest/transform/robustness.py:960]: it should be mandatory below line:
 for idx, sample in enumerate(sample_list):

Comment (chakravarthik27) [reply nlptest/transform/robustness.py:960]: and write a code accordingly to uppercase or other test classes
bcoz, it handles both samples and strings from a list.

**Review by chakravarthik27 — CHANGES_REQUESTED** [review]

**Comment by Prikshit7766** [comment]
number to word and ocr typo working fine

**Pull request merged**